specification/mixedreality/resource-manager/Microsoft.MixedReality/preview/2019-02-28-preview/mixedreality.json

```
@@ -0,0 +1,724 @@
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MixedReality/spatialAnchorsAccounts/{spatialAnchorsAccountName}/keys" : {
+      "get": {
+        "operationId": "GetKeys",
+        "tags": [
+          "Key"
+        ],
+        "description": "Get Both of the 2 Keys of a Spatial Anchors Account",
+        "x-ms-examples": {
+          "ResourceRegenerateKey": { "$ref": "./examples/SpatialAnchorsAccount/GetKeys.json" }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/spatialAnchorsAccountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SpatialAnchorsAccountKeys"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "post": {
+        "operationId": "RegenerateKeys",
+        "tags": [
+          "Key"
+        ],
+        "description": "Regenrate 1 Key of a Spatial Anchors Account",
+        "x-ms-examples": {
+          "ResourceRegenerateKey": { "$ref": "./examples/SpatialAnchorsAccount/RegenerateKey.json" }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/spatialAnchorsAccountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/spatialAnchorsAccountKeyRegenerateParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SpatialAnchorsAccountKeys"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions" : {
+    "CheckNameAvailabilityRequest" : {
+      "description": "Check Name Availability Request",
+      "type": "object",
+      "required" : ["name", "type"],
+      "properties": {
+        "name": {
+          "description": "Resource Name To Verify",
+          "type": "string"
+        },
+        "type": {
+          "description": "fully qualified resource type which includes provider namespace",
```

Descriptions should be in complete sentences.
```suggestion
          "description": "Fully qualified resource type which includes provider namespace.",
```